app/app_stack.py

```diff
@@ -129,13 +207,44 @@ def __init__(self, scope: Construct, id: str, **kwargs) -> None:
         )
         checkfile_exists_state.when(
             sfn.Condition.number_equals("$.CheckFile.Payload.StatusCode", 404),
-            download_era5_task,
+            send_request_task,
         )
         checkfile_exists_state.otherwise(check_file_fail_state)
+
+        wait_state = sfn.Wait(
+            self, "Wait", time=sfn.WaitTime.duration(Duration.seconds(10))
```

Comment:
How did you conclude this was a good number? Seems a little on the short side to me.

Reply:
The standard cdsapi client starts with 1 second and for each try it multiplies with 1.5 until it reaches 120s, it retries 500 times.

Our first try completed after the first wait-state, obviously no load on the service at that time. Something like 10s - 30s should be reasonable.